grants/casper_stats/m1.md

```diff
@@ -0,0 +1,21 @@
+---
+Milestone: "M1",
+Status: "In Review",
+---
+<!--lang:en--> 
+### Description
+
+1. As first step in the integration of Casper Network we are required by the regulators to do an
+extensive token DD, which will investigated any potential bad media, court rullings, and team’s
```

Comment:
A couple of typos to address please;
1. investigated instead of investigate
2. $CSRP instead of $CSPR
3. Blockahin instead of Blockchain

Reply:
Also, Casper Stats milestone details contain details about Ramp Network or vice versa. Please check and update as you see fit.